models/stg_klaviyo__person.sql

```diff
@@ -42,6 +42,7 @@ final as (
         timezone,
         title,
         updated as updated_at,
+        last_event_date as last_event_at,
```

Comment:
ah actually this is causing ambiguous column errors in the final persons model due to https://github.com/fivetran/dbt_klaviyo/blob/main/models/intermediate/int_klaviyo__person_metrics.sql#L17

may want to leave it as `last_event_date` or consider removing it?

Reply:
Oh good noticing, I didn't realize the downstream error. I feel like it's still worthwhile adding it, so will just keep it as `last_event_date`. Thanks Jamie!